csb/src/main/scala/edu/msstate/dasi/csb/OptionParser.scala

```diff
@@ -34,26 +34,15 @@ class OptionParser(override val programName: String, programVersion: String, con
   note("")
 
   cmd("seed")
-    .text("Generates a property graph and the probability distributions of its properties starting from two log files.")
+    .text("Generates a property graph and the probability distributions of its properties starting from a bro log file.")
     .action( (_, c) => c.copy(mode = "seed") )
     .children(
-      opt[String]('a', "alert-log")
-        .valueName("<path>")
-        .text(s"Path of the Snort connection log [default: ${config.alertLog}].")
-        .validate(path => if ( new File(path).isFile ) success else failure(s"$path is not a regular file") )
-        .action((x, c) => c.copy(alertLog = x)),
-
       opt[String]('b', "bro-log")
```

Comment:
Since we discarded the snort and the augLog options, this option can be changed to:
````scala
 opt[String]('l', "log")
````
as it would be easier to use.